src/cargo/ops/cargo_new.rs

```diff
@@ -759,69 +759,53 @@ fn mk(config: &Config, opts: &MkOptions<'_>) -> CargoResult<()> {
     init_vcs(path, vcs, config)?;
     write_ignore_file(path, &ignore, vcs)?;
 
-    let mut cargotoml_path_specifier = String::new();
+    // Create `Cargo.toml` file with necessary `[lib]` and `[[bin]]` sections, if needed.
+    let mut manifest = toml_edit::Document::new();
```

Comment:
Tests are failing on this commit.

Please make sure all commits pass tests as its important for us to see what the effect is of a change.  In this case, I would consider the new output a regression

e.g.
```
---- expected: tests/testsuite/init/lib_already_exists_nosrc/out/Cargo.toml
++++ actual:   /home/epage/src/personal/cargo/target/tmp/cit/t1374/case/Cargo.toml
        1 + lib = { name = "case", path = "lib.rs" }
        2 +
   1    3 | [package]
   2    4 | name = "case"
   3    5 | version = "0.1.0"
   4    6 | edition = "2021"
   5    7 |
   6    8 | # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
   7    9 |
   8   10 | [dependencies]
   9      -
  10      - [lib]
  11      - name = "case"
  12      - path = "lib.rs"
tests/testsuite/init/lib_already_exists_nosrc/out/lib.rs: is good
```

Reply:
Fixed.